NorthwindCRUD.Tests/CustomerServiceFixture.cs

```diff
@@ -0,0 +1,36 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace NorthwindCRUD.Tests
+{
+    [TestClass]
+    public class CustomerServiceFixture : BaseFixture
+    {
+        [TestMethod]
+        public void ShouldCreateCustomer()
+        {
+            var customer = DataHelper.GetCustomer();
+            var createdCustomer = DataHelper.CustomerService.Create(customer);
+
+            Assert.IsNotNull(createdCustomer);
```

Comment:
Do we know if `customer !== createdCustomer`?
If they're the same exact instance then every other assert is useless.

In App Builder tests we had to create separate contexts to retrieve new instances of the object.

Reply:
Good catch, I added a second DataHelper that goes against a new datacontext in order to do the asserts.